naucse/templates/lesson.html

```diff
@@ -1,7 +1,7 @@
 {% extends "_base.html" %}
 
 {% block style %}
-    {% if page.css %}
+    {% if page.css is defined and page.css %}
```

Comment:
I think a separate template for cover pages would be better. The common things can be shared via template inheritance.